service/app/controllers/coursemanagement/validator/CourseBatchRequestValidator.java

```diff
@@ -45,10 +45,19 @@ public void validateUpdateCourseBatchRequest(Request request) {
             ERROR_CODE);
       }
     }
-    validateParam(JsonKey.NAME, ResponseCode.mandatoryParamsMissing, JsonKey.NAME);
+    if (request.getRequest().containsKey(JsonKey.NAME)) {
+      validateParam(
+          (String) request.getRequest().get(JsonKey.NAME),
+          ResponseCode.mandatoryParamsMissing,
```

Comment:
This will confuse user -> update should not expect mandatory parameters. 
Error Code should be cannotUnsetMandatoryParam

Message could be - Cannot unset mandatory parameter {0}

Reply:
Discuss with @manzarul and remove this check altogether.